src/commondao/common.dao.ts

```diff
@@ -728,28 +728,24 @@ export class CommonDao<BM extends BaseDBEntity, DBM extends BaseDBEntity = BM, I
       }
       Object.assign(bm, patch)
     } else {
-      const loaded = await this.getById(bm.id as ID, {
+      const loaded = await this.requireById(bm.id as ID, {
```

Comment:
p333: we **might** benefit from a dedicated error message for this case, so developer can understand that it was a `patch` that threw, not some `requireById` that they didn't write.

But I'm not 100% convinced it's worth extra code. But maybe 80% convinced, it's just 1 assert line with some metadata.

Reply:
On the other hand, semantically - it's the same as `requireById` error - the required object did not exist.

If I think about the end-user (not developer), who might peek at the nerdy error details (if we ever share those) - the more generic error from requireById is actually better than more-specific error mentioning some `patch`.